### ISSUE TITLE
Resetting A Domain's Engine Name Doesn't Reset Hostname in Database

### ISSUE THREAD

**Issue by rightker** [OpenMake-Software/ReleaseEngineer#32]
A new Domain has the value "Deployment Engine" in its Engine field in the General tab. Also, the hostname of the server where the engine is installed is contained in the dm_engine table in the database. If the value in the Engine field within the General tab for a Domain is changed to the hostname of a different server, the dm_engine table's hostname field is changed to reflect this. The problem is, if the Engine field is changed back to "Deployment Engine", the value of the hostname is not changed back to the default value.

**Comment by pgibbs-openmake** [comment]
No, it's not. This is a bit clunky but we should aim to do something with this for a later release (perhaps by having an engine tab where hostname, threads etc can be set).
